pyomo/dae/diffvar.py

```diff
@@ -132,7 +131,7 @@ def __init__(self, sVar, **kwds):
                     "The variable %s is indexed by multiple ContinuousSets. "
                     "The desired ContinuousSet must be specified using the "
                     "keyword argument 'wrt'" % sVar)
-            wrt = [next(iterkeys(sVar._contset)), ]
+            wrt = [next(iter(sVar._contset.keys())), ]
```

Comment:
This can probably be simplified, let me take a closer look.

Reply:
FWIW, I tried some simpler ideas and nothing I could come up with worked...